src/us/kbase/narrativejobservice/sdkjobs/DockerRunner.java

```diff
@@ -106,9 +106,14 @@ public File run(
             }
             cntCmd = cntCmd.withEnv(envVarList.toArray(new String[envVarList.size()]));
 
-            if(! System.getenv("KB_DOCKER_NETWORK").equals("")) {
-                cntCmd.withNetworkMode(System.getenv("KB_DOCKER_NETWORK"));
-            }
+//            try {
```

Comment:
Are we sure this is no longer required?

Reply:
Yes this  is for mini kb

Reply:
Looks good.

Reply:
Do you mean this was added for mini kb?  I think I use this when I run at NERSC.  I don't think we should just drop this.

Reply:
@sychan We have a docker network called mini_kb inside mini_kb. This code is there to be uncommented when doing testing in mini_kb, and I add the ENV variable called KB_DOCKER_NETWORK to the localmethodrunner.sh file.

Reply:
I see - so you're just checking to see if it is running in a docker network at all. Depending on future deployment models, it is not out of the question that hosts might run in a docker overlay network. But we can burn that bridge when we come to it, I suppose.